qiskit/quantum_info/states/statevector.py

```diff
@@ -278,13 +278,14 @@ def from_label(cls, label):
             of the label is larger than an explicitly specified num_qubits.
 
         Additional Information:
-            The labels correspond to the single-qubit states:
-            '0': [1, 0]
-            '1': [0, 1]
-            '+': [1 / sqrt(2), 1 / sqrt(2)]
-            '-': [1 / sqrt(2), -1 / sqrt(2)]
-            'r': [1 / sqrt(2), 1j / sqrt(2)]
-            'l': [1 / sqrt(2), -1j / sqrt(2)]
+
+        The labels correspond to the single-qubit states:
```

Comment:
This might appear as a weird heading in the docs because of the `:`

Reply:
Looking at the output from the azure docs job, it looks awful: 
![Screenshot_2020-03-04_14-46-42](https://user-images.githubusercontent.com/2447371/75916677-c7ad1a00-5e26-11ea-8229-920b88ae1603.png)

But it didn't treat it as a heading. I will use the list table though, that will definitely look better

Reply:
Adjusted in: https://github.com/Qiskit/qiskit-terra/pull/3902/commits/5f8aff5cd0aa64212b126c9a33d70fa15c9a58bd